tests/util/test_check_dependencies.py

```diff
@@ -67,6 +68,18 @@ def test_mandatory_dependency(self) -> None:
                 # should not raise
                 check_requirements()
 
+    def test_version_reported_as_None(self) -> None:
```

Comment:
naming convention suggestion/nit: the capital letter is unusual!

```suggestion
    def test_version_reported_as_none(self) -> None:
```